src/components/Tabs/Tabs.module.css

```diff
@@ -34,7 +34,11 @@
 }
 
 .tabs--scrollable-right {
-  -webkit-mask-image: -webkit-linear-gradient(right, transparent);
+  -webkit-mask-image: -webkit-linear-gradient(
+    right,
+    transparent,
+    white var(--eds-size-8)
```

Comment:
Could you help me explain what this changes exactly? I can't tell the difference in Chrome.

Reply:
Re introduces the right masking when there is no more left content but there is more right content (also in chrome):

Currently:
![Screen Shot 2022-10-12 at 12 52 49 PM](https://user-images.githubusercontent.com/86632227/195435210-038caf5c-a639-49df-afef-12a785dfe052.png)

This change:
![Screen Shot 2022-10-12 at 12 52 47 PM](https://user-images.githubusercontent.com/86632227/195435204-98313014-97a8-4583-918e-6f9adb39ddc5.png)

Reply:
Awesome, thanks!